src/components/Context/timestampFactory.js

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default dateProvider => {
+  return event => {
+    const date = dateProvider();
+    event.setTimestamp(date.toISOString());
```

Comment:
I'm wondering if this needs to be local as well. The examples I see in github seem to show the local variant:

https://github.com/adobe/xdm/blob/76f7077ed60c3e282635abddee9f78f7b9d878df/schemas/context/experienceevent.example.1.json#L7

Reply:
If you look in that example on Line 83, the localTime has timezone +13:00 whereas the timestamp has +00:00; however, the time is the same.  Every example has the timestamp in +00:00

Reply:
Okay, I wasn't as confident that `+00:00` means it should always be `+00:00`. I thought they may have just not been creative with the timezone when creating the example, but I think you're probably right that it's always `+00:00`.